docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
     - Decision tree point
-    - 
+    - Age, scenario 
+    - Decision tree value & export value 
   * - Ultrasound status (AI assisted, standard, none)
     - Decision tree point
-    - 
+    - Scenario
+    - Decision tree value & export value 
   * - Gestational age at birth
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
-    - 
+    - Decision tree value & export value 
+    - Scenario
```

Comment:
These lines are flipped!

Reply:
I think gestational age (LBWSG) will be age group specific? So we should include that as a stratification. But if LBWSG is not age specific ignore this comment.

Reply:
Do you mean LBWSG exposure would be dependent on the age of the birthing parent? It seems like that would be true in real life, but GBD models LBWSG in isolation, without connecting it to information about the birthing parent, so we don't have that data. Or did you mean something else?